.github/workflows/build.yml

```diff
@@ -227,26 +224,33 @@ jobs:
 
     steps:
       - name: Checkout
-        uses: actions/checkout@v2
+        uses: actions/checkout@v2.4.0
```

Comment:
there's no reason to pin a specific version also this should be `v3`

Reply:
Weird, I didn't realize v3 was a thing. Seems to work